src/main/java/com/hotjava/app/hotjava/service/VehicleService.java

```diff
@@ -88,25 +97,25 @@ public void saveImage(MultipartFile imageFile, Photo photo) throws IOException {
      * @return boolean value for if the vehicle's is considered to be completely filled out
      */
     private boolean vehicleIsComplete(Vehicle vehicle) {
-        if(vehicle.getVehicleSubmissionID() == 0){
+        if(vehicle.getSubmissionID() == 0){
             return false;
         }
-        if(vehicle.getVehicleMake().equals(null) || vehicle.getVehicleMake().isEmpty()){
+        if(vehicle.getMake().equals(null) || vehicle.getMake().isEmpty()){
             return false;
         }
-        if(vehicle.getVehicleModel().equals(null) || vehicle.getVehicleModel().isEmpty()){
+        if(vehicle.getModel().equals(null) || vehicle.getModel().isEmpty()){
             return false;
         }
-        if(vehicle.getVehicleScore().equals(null) || vehicle.getVehicleScore().isEmpty()){
+        if(vehicle.getScore().equals(null) || vehicle.getScore().isEmpty()){
             return false;
         }
-        if(vehicle.getVehicleYear().equals(null) || vehicle.getVehicleYear().isEmpty()){
+        if(vehicle.getYear().equals(null) || vehicle.getYear().isEmpty()){
             return false;
         }
-        if(vehicle.getVehicleDescription().equals(null) || vehicle.getVehicleDescription().isEmpty()){
+        if(vehicle.getDescription().equals(null) || vehicle.getDescription().isEmpty()){
             return false;
         }
-        if(vehicle.getVehicleOwnerName().equals(null) || vehicle.getVehicleOwnerName().isEmpty()){
+        if(vehicle.getOwnerName().equals(null) || vehicle.getOwnerName().isEmpty()){
```

Comment:
Removed redundant 'vehicle' in variable names as it is implied these belong to a vehicle.

Reply:
Same as above, implemented with Camel Case